docs/src/main/asciidoc/continuous-testing.adoc

```diff
@@ -0,0 +1,146 @@
+----
+
+
+NOTE: If you want continuous testing to start automatically you can set `quarkus.test.continuous-testing=enabled` in
+`application.properties`. If you don't want it at all you can change this to `disabled`.
+
+
+Now you can start making changes to your application. Go into the `GreetingResource` and change the hello endpoint to
+return `"hello world"`, and save the file. Quarkus should immediately re-run the test, and you should get output similar
+to the following:
+
+----
+2021-05-11 14:21:34,338 ERROR [io.qua.test] (Test runner thread) Test GreetingResourceTest#testHelloEndpoint() failed
+: java.lang.AssertionError: 1 expectation failed.
+Response body doesn't match expectation.
+Expected: is "hello"
+  Actual: hello world
+
+	at io.restassured.internal.ValidatableResponseImpl.body(ValidatableResponseImpl.groovy)
+	at org.acme.getting.started.GreetingResourceTest.testHelloEndpoint(GreetingResourceTest.java:21)
+
+
+--
+Test run failed, 2 tests were run, 1 failed, 0 were skipped. Tests took 295ms
+Press [r] to re-run, [v] to view full results, [p] to pause, [h] for more options>
+----
+
+Change it back and the tests will run again.
+
+== Controlling Continuous Testing
+
+There are various hotkeys you can use to control continuous testing. Pressing `h` will display the following list
+of commands:
+
+----
+[r] - Re-run all tests
+[f] - Re-run failed tests
+[b] - Toggle 'broken only' mode, where only failing tests are run
+[v] - Print failures from the last test run
+[o] - Toggle test output
+[p] - Pause tests
+[i] - Toggle instrumentation based reload
+[l] - Toggle live reload
+[s] - Force live reload scan
+[h] - Display this help
+----
+
+These are explained below:
+
+[r] - Re-run all tests::
+This will re-run every test
+
+[f] - Re-run failed tests::
+This will re-run every failing test
+
+[b] - Toggle 'broken only' mode, where only failing tests are run::
+Broken only mode will only run tests that have previously failed, even if other tests would normally be affected by a code
+change. This can be useful if you are modifying code that is used by lots of tests, but you only want to focus on debugging
+the failing one.
+
+[v] - Print failures from the last test run::
+Prints the failures to the console again, this can be useful if there has been lots of console output since the last run.
+
+[o] - Toggle test output::
+By default test output is filtered and not displayed on the console, so that test output and dev mode output is not
+interleaved. Enabling test output will print output to the console when tests are run. Even when output is disabled
+the filtered output is saved and can be viewed in the Dev UI.
+
+[p] - Pause tests::
+Temporarily stops running tests. This can be useful if you are making lots of changes, and don't want feedback until they
+are all done.
+
+[i] - Toggle instrumentation based reload::
+This is not directly related to testing, but allows you to toggle instrumentation based reload. This will allow dev mode
```

Comment:
```suggestion
This is not directly related to testing, but allows you to toggle instrumentation based reload. This will allow live reload
```
You use live reload in the other items so better be consistent.